src/app/components/SubjectHeading/Search/SubjectHeadingSearch.jsx

```diff
@@ -38,15 +38,16 @@ class SubjectHeadingSearch extends React.Component {
 
   makeApiCallWithThrottle(timerId) {
     const apiCall = () => {
-      if (this.state.userInput.length) return axios(`${appConfig.baseUrl}/api/subjectHeadings/autosuggest?query=${this.state.userInput}`)
+      if (this.state.userInput) return axios(`${appConfig.baseUrl}/api/subjectHeadings/autosuggest?query=${this.state.userInput}`)
         .then((res) => {
           if (res.data.request.query.trim() === this.state.userInput.trim()) {
             this.setState({
               suggestions: res.data.autosuggest,
               activeSuggestion: 0,
             });
           }
-        });
+        })
+        .catch(console.error);
```

Comment:
Is this the same thing as `x => console.error(x)` ? If so that is nice.
Has this been tested manually?

Reply:
Yes that is how that short hand works. Seems to console log with or without that catch, but best practices to have it and potentially there could be other logic we want to put in there later.